### DISCUSSION TITLE
ListBox should be controlled by a parent component

### DISCUSSION THREAD

**Discussion by daniele-bolla** [tailwindlabs/headlessui#760] [General]
Same issue as in https://github.com/tailwindlabs/headlessui/discussions/633.

I would like to use ListBox an wrap it with a custom BaseSelect component.
How could i handle the modelValue changes?

The component doesn't seem to emit any "cahnge" or "input" event.

**Comment by tangye1234** [comment]
Or, maybe it can provide a close(focusElement?: HTMLElement | Ref<HTMLElement>) as a prop in the child function's argument
